code/modules/reagents/reagent_containers/dropper.dm

```diff
@@ -9,13 +9,21 @@
 	amount_per_transfer_from_this = 5
 	possible_transfer_amounts = list(1,2,3,4,5)
 	volume = 5
-	var/filled = 0
+
+/obj/item/weapon/reagent_containers/dropper/on_reagent_change()

```

Comment:
consistency with syringes and others, bottles do the exact same thing https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/reagents/reagent_containers/bottle.dm#L15:L44